src/env.d/06-chromebrew

```diff
@@ -2,6 +2,8 @@
 
 # Do not edit this file. It will be overwritten by updates.
 
+shopt -s extglob
```

Comment:
`env.d` doesn't seem to be the location where Bash-specific configurations should be put. Shouldn't Bash-specific configurations go into some files in `bash.d`?

In the first place, I think we don't have to set `extglob` for `bash-completion` because `bash-completion` sets `extglob`. My initial guess that "*`extglob` might be turned off somewhere after loading `bash-completion`*" was false. It was just caused by the loading of the exported functions, which is performed independently of the initialization of `bash-completion`.

If you would like to turn on `extglob` regardless of `bash-completion`, you can just put `extglob` somewhere in `bash.d` (without caring the loading order with respect to `bash-completion`).